ruby/test/integration/grind_redis_test.rb

```diff
@@ -256,6 +259,7 @@ def test_grind_max_test_duration_failing
           '--worker', '1',
           '--timeout', '5',
           '--max-test-duration', '0.00001',
+          '--track-execution-duration',
```

Comment:
Since the other flag is called `max-test-duration`, wouldn't it be more consistent to call this flag `track-test-duration`?